docs/source/en/perf_train_gpu_one.md

```diff
@@ -311,13 +318,6 @@ adam_bnb_optim = bnb.optim.Adam8bit(
 )
 ```
 
-<Tip>
```

Comment:
Can you elaborate why this has been removed?

Reply:
I don't believe this is needed anymore. If I'm not mistaken, this should no longer be necessary because of the changes in this PR: https://github.com/huggingface/transformers/pull/15622/

Referencing this discussion on the specific change here: https://github.com/huggingface/transformers/pull/15622/files#r804948681

Reply:
Makes sense, thanks a lot!